6/README.md

```diff
@@ -205,45 +228,110 @@ The endpoint must accept
 - POST requests to the `/async/operation-id` (e.g. https://service-endpoint/async/4d517500da0acb0d65a716f61330969334630363ce4a6a9d39691026ac7908ea) along with JSON formatted payload as described by the params section of the operation metadata.
 
 - The path argument operation-id must be the ID of the operation asset.
-- The keys in the (payload) map must be parameter names as specified in the operation metadata.
-- All required parameters must be included.
-
-- The values must be one of 
-  - A map (if the type is **asset**).
-    - The map must contain the DID (the Decentralized ID) against the key **did** 
-  - A valid JSON value (if the type is **json**). Here's the list of [valid JSON data types](https://json-schema.org/understanding-json-schema/reference/type.html)
+- The keys in the (payload) map may be parameter names as specified in the operation metadata.
+- All required parameters may be included.
+
+- The values can be any of following:
+   OpenAPI SchemaObject [OpenAPI Schema Object standards](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#schema-object).
```

Comment:
This appears too heavyweight a schema design to make a mandatory part of the standard. Let's revert to `json` and `asset`for now.